<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> JsonDog</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1000</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>3000</width>
          <height>40</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnInsert">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>插入同级</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnInsertChild">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>插入子级</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemove">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>删除结构</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1000</width>
          <height>500</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QTabWidget" name="tabWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="file1">
              <attribute name="title">
               <string> 文件</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QTableView" name="filetableView"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="struct1">
              <attribute name="title">
               <string> 结构</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QTreeView" name="treeView"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4,1">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QTextEdit" name="textEdit">
                  <property name="palette">
                   <palette>
                    <active>
                     <colorrole role="Button">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>30</red>
                        <green>30</green>
                        <blue>30</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="BrightText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>208</red>
                        <green>208</green>
                        <blue>208</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Base">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>30</red>
                        <green>30</green>
                        <blue>30</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Window">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>30</red>
                        <green>30</green>
                        <blue>30</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Highlight">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>0</green>
                        <blue>0</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="HighlightedText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>208</red>
                        <green>208</green>
                        <blue>208</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="PlaceholderText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="128">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </active>
                    <inactive>
                     <colorrole role="Button">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>30</red>
                        <green>30</green>
                        <blue>30</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="BrightText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>208</red>
                        <green>208</green>
                        <blue>208</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Base">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>30</red>
                        <green>30</green>
                        <blue>30</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Window">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>30</red>
                        <green>30</green>
                        <blue>30</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Highlight">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>0</green>
                        <blue>0</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="HighlightedText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>208</red>
                        <green>208</green>
                        <blue>208</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="PlaceholderText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="128">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </inactive>
                    <disabled>
                     <colorrole role="Button">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>30</red>
                        <green>30</green>
                        <blue>30</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>120</red>
                        <green>120</green>
                        <blue>120</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="BrightText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>208</red>
                        <green>208</green>
                        <blue>208</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Base">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>30</red>
                        <green>30</green>
                        <blue>30</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Window">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>30</red>
                        <green>30</green>
                        <blue>30</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Highlight">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>120</green>
                        <blue>215</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="HighlightedText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>208</red>
                        <green>208</green>
                        <blue>208</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="PlaceholderText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="128">
                        <red>0</red>
                        <green>0</green>
                        <blue>0</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </disabled>
                   </palette>
                  </property>
                  <property name="font">
                   <font>
                    <family>微软雅黑</family>
                    <pointsize>18</pointsize>
                    <stylestrategy>NoAntialias</stylestrategy>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color:rgb(30,30,30)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextBrowser" name="textBrowser">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1206</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionnew"/>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actioncheck"/>
    <addaction name="actionbuildtree"/>
    <addaction name="actionbuildcode"/>
    <addaction name="actionrefresh"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>云服务</string>
    </property>
    <addaction name="action1"/>
    <addaction name="action2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionnew">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionsaveas">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actioncheck">
   <property name="text">
    <string>检查语法</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionbuildtree">
   <property name="text">
    <string>生成树形图</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionbuildcode">
   <property name="text">
    <string>生成代码</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>添加到云端</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>从云端获取</string>
   </property>
  </action>
  <action name="actionrefresh">
   <property name="text">
    <string>刷新文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
